packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -103,6 +119,12 @@ export class BaseButton extends BaseComponent<IButtonProps, {}> implements IButt
   }
 
   protected onRenderContent(tag: any, buttonProps: IButtonProps): JSX.Element {
+    let {
```

Comment:
simiilar to autobind discussion, I don't believe we've been using 'const' for setting props in previous components. I agree that const is probably more correct, but if I change these, I should probably change ALL of the prop declarations in this file, and set this as a standard moving forward. @dzearing ?